packages/@okta/i18n/src/properties/login.properties

```diff
@@ -841,6 +841,8 @@ oie.on_prem.verify.passcode.label = Enter code
 oie.numberchallenge.warning = Haven't received a push notification yet? Try opening the Okta Verify app on your device, or <$1>resend the push notification</$1>.
 ## {0} refers to a number which is the correct answer for the number challenge
 oie.numberchallenge.instruction = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
+idx.authenticator.app.method.push.force.upgrade.number_challenge.title = Update Okta Verify
+idx.authenticator.app.method.push.force.upgrade.number_challenge = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifcations, update Okta Verify to the latest version, then try again.
```

Comment:
Can we prefix these keys with `oie` instead to align with the other authenticator labels?

Reply:
The `idx.authenticator.app.method.push.force.upgrade.number_challenge` key is actually coming from the server. I went with `idx.authenticator.app.method.push.force.upgrade.number_challenge.title` to keep them consistent.

Reply:
Ah - this is interesting.

Can we add a comment above these keys to mention that they're tied to a server response versus a client configuration?

Reply:
I can update `idx.authenticator.app.method.push.force.upgrade.number_challenge.title` to `oie.numberchallenge.force.upgrade.title`